### PR TITLE
attempt to finetune some details

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
greetings everyone

on behalf of my own, i would like to suggest some changes to sinosc helpfile. my main goal within suggesting these changes is concerned with fine tunning small details related to some grammar checks. hope this pull request is meaningful to supercollider development and to supercollider community. if it isn't, simply withdraw my changes. i give you guys the option of editing my changes, so that we can actually speed up the process of going throughout this. feel free to draw some harsh criticism if necessary. made with love. peace. tiago

### CODE REVIEW

File: HelpSource/Classes/SinOsc.schelp [hunk 1]
```diff
@@ -7,15 +7,16 @@ categories::  UGens>Generators>Deterministic
 Description::
 
 Generates a sine wave.
-Uses a wavetable lookup oscillator with linear interpolation.
-Frequency and phase modulation are provided for audio-rate modulation.
-Technically, code::SinOsc:: uses the same implementation as  link::Classes/Osc::  except that its table is fixed to be a sine wave made of code::8192:: samples.
+Uses a wavetable oscillator with linear interpolation.
+Frequency and phase modulaton is possible on the audio rate domain.
```
Comment (danstowell) [inline HelpSource/Classes/SinOsc.schelp]: I'm afraid "is possible" should be "are possible" (i.e. this edit is grammatically inappropriate). Also I would not say "on the audio rate domain" - audio rate is not a domain. Sorry but I'd reject this edit.

File: HelpSource/Classes/SinOsc.schelp [hunk 2]
```diff
@@ -7,15 +7,16 @@ categories::  UGens>Generators>Deterministic
...
+code::SinOsc:: uses the implementation of  link::Classes/Osc::.  
+Its table is fixed; its waveshape is set to code::8192:: samples.
```
Comment (danstowell) [inline HelpSource/Classes/SinOsc.schelp]: I'd vote to reject this edit - it is shorter, but for me it seems less clear.

File: HelpSource/Classes/SinOsc.schelp [hunk 3]
```diff
@@ -46,18 +47,20 @@ Examples::
...
+// audio-rate sine wave at 200 Hz,
+// starting with phase 0 and amplitude of 0.5
 { SinOsc.ar(200, 0, 0.5) }.play;
 
-// modulate the frequency with an exponential ramp
+// frequency modulation with an exponential ramp
 { SinOsc.ar(XLine.kr(2000, 200), 0, 0.5) }.play;
 
-// more complex frequency modulatation
+// more on frequency modulation
```
Comment (danstowell) [inline HelpSource/Classes/SinOsc.schelp]: Good catch "modulatation". However I would stick with "more complex"

File: HelpSource/Classes/SinOsc.schelp [hunk 4]
```diff
@@ -46,18 +47,20 @@ Examples::
...
 { SinOsc.ar(SinOsc.ar(XLine.kr(1, 1000, 9), 0, 200, 800), 0, 0.25) }.play;
 
-// phase modulation (see also PMOsc)
+// phase modulation (see PMOsc)
```
Comment (danstowell) [inline HelpSource/Classes/SinOsc.schelp]: no, "see also" is more appropriate

**Comment by danstowell** [comment]
I'm sorry, but as a native English speaker I would say this pull request does not improve the language. It introduces some errors. Thank you for spotting "modulatation" though

**Comment by tiagmoraismorgado** [comment]
i welcome your feedback, and i see it as valid. perharps we should also discuss this with others. at least taking advantage of some form of changes, in the tiny little details. i am sure there is something relevant in the midst of all my commitments within this post, and there might be something relevant for us to help improving supercollider. counting on you to help checking this ;)

**Comment by tiagmoraismorgado** [comment]
and furthermore i allowed edits, so if you guys agree, we can keep this open, edit the details collectively, as a team, and coming up with something better, in the end

**Comment by danstowell** [comment]
Thanks for taking the feedback kindly.
If I was "the editor" my decision would be to accept only the one typo fix and nothing else. It takes time to go through requests like this one and you'll probably find not many people wanting to collaborate on this PR because of the time required. I could be wrong though...

**Comment by vivid-synth** [comment]
Agreed, @tiagmoraismorgado can you make a separate pull request for the "modulatation" typo fix? Thanks!

**Pull request closed without merging**